### DISCUSSION TITLE
Third party UI package with yarn PnP (NextJS project)

### DISCUSSION THREAD

**Discussion by ybelakov** [tailwindlabs/tailwindcss#11118] [Help]
Hello,guys, I am working on a NextJS typescript project with tailwindcss. I am having hard time integrating a third-party library in the project (just like preline, tailwindui and etc). The problems arises from including the package in the `content` array in the `tailwind.config.js`.  Example from the `preline` package installation guide.

```
module.exports = {
  content: [
      'node_modules/preline/dist/*.js',
  ],
  plugins: [
      require('preline/plugin'),
  ],
}
```

Since I am using yarn PnP, there is no `node_modules` folder, so I am wondering how I can specify the package in the tailwind config file. I will be very thankful if someone manages to help me out.

**Comment by lxia1220** [comment]
@ybelakov I'm not sure this is the best way, but it works anyway.

```js
const nexUiThemePath = path.dirname(require.resolve("@nextui-org/theme"));

module.exports = {
  content: [
    `${nexUiThemePath}/**/*.{js,ts,jsx,tsx}`,
  ],
};
```
